ios/RNCSafeAreaViewEdgeModes.m

```diff
@@ -0,0 +1,29 @@
+#import "RNCSafeAreaViewEdgeModes.h"
+#import <React/RCTConvert.h>
+
+@implementation RCTConvert (RNCSafeAreaView)
+
+RCT_MULTI_ENUM_CONVERTER(
+    RNCSafeAreaViewEdgeModes,
+    (@{
+      @"off" : @(RNCSafeAreaViewEdgeModesOff),
+      @"additive" : @(RNCSafeAreaViewEdgeModesAdditive),
+      @"maximum" : @(RNCSafeAreaViewEdgeModesMaximum),
+    }),
+    RNCSafeAreaViewEdgeModesOff,
+    integerValue);
+
+@end
+
+@implementation NSString (EnumParser)
+
+- (RNCSafeAreaViewEdgeModes)RNCSafeAreaViewEdgeModesFromString{
+    NSDictionary<NSString*,NSNumber*> *edgeModes = @{
+                            @"off": @(RNCSafeAreaViewEdgeModesOff),
+                            @"additive": @(RNCSafeAreaViewEdgeModesAdditive),
+                            @"maximum": @(RNCSafeAreaViewEdgeModesMaximum),
+                            };
+    return edgeModes[self].integerValue;
```

Comment:
This is what RCT_ENUM_CONVERTER does - with additional logic to avoid recreating the dict etc.

Reply:
Yeah I know, but I just can't get it to work with RCTConvert. It is killing me. Having `RNCSafeAreaViewEdgeModeFromString` was the only way it worked.  I can get rid of 

```
RCT_ENUM_CONVERTER(
    RNCSafeAreaViewEdgeMode,
    (@{
      @"off" : @(RNCSafeAreaViewEdgeModeOff),
      @"additive" : @(RNCSafeAreaViewEdgeModeAdditive),
      @"maximum" : @(RNCSafeAreaViewEdgeModeMaximum),
    }),
    RNCSafeAreaViewEdgeModeOff,
    integerValue);
    
```
    
 Since that `RNCSafeAreaViewEdgeModeFromString` does the same thing int RCT_CUSTOM_CONVERTER for `RNCSafeAreaViewEdges` and we never pass `RNCSafeAreaViewEdgeMode` from JS directly

Reply:
Added a comment I _think_ is the issue you're facing